lib/lrama/binding.rb

```diff
@@ -0,0 +1,23 @@
+module Lrama
+  class Binding
```

Comment:
`Binding` is only used when building actual rules from grammar file, then I think it's better to be placed under `Grammar` namespace.

Reply:
Thank you. You are indeed correct, so I have updated this PR.